docs/index.md

```diff
@@ -3,7 +3,7 @@ title: "Introduction"
 weight: 10
 ---
 
-The AsyncAPI generator is a tool that generates anything you want using the **[AsyncAPI Document](asyncapi-document.md)** and **[Template](template.md)** that are supplied as inputs to the AsyncAPI CLI. The generator was built with extensibility in mind; you can use the generator to generate anything you want, provided that it can be defined in a template, such as code, diagrams, markdown files, microservices, and applications. A number of [community-maintained templates](https://github.com/search?q=topic%3Aasyncapi+topic%3Agenerator+topic%3Atemplate) are now available for immediate usage.
+The AsyncAPI generator is a tool that generates anything you want using the **[AsyncAPI Document](generator/asyncapi-documents)** and **[Template](generator/template)** that are supplied as inputs to the AsyncAPI CLI. The generator was built with extensibility in mind; you can use the generator to generate anything you want, provided that it can be defined in a template, such as code, diagrams, markdown files, microservices, and applications. A number of [community-maintained templates](https://github.com/search?q=topic%3Aasyncapi+topic%3Agenerator+topic%3Atemplate) are now available for immediate usage.
```

Comment:
why `(generator/asyncapi-documents)` and not `(asyncapi-documents)` like with links in other documents?

I mean in some documents you make links with `generator/` prefix, and in some without it? is there a reason? All these documents display on the same level so logically for me they should follow the same pattern

Reply:
While doing manual testing on the website, the ones with the generator/ suffix were breaking without it.